src/config.ts

```diff
@@ -58,7 +58,7 @@ export async function loadConfig(configFilename: string = CONFIG_FILENAME): Prom
 
   log('Resolved config file.');
 
-  const loaded = mergeDeepRight(DEFAULT_CONFIG, loadedConfig) as Configuration;
+  const loaded = mergeDeepRight<Configuration, Configuration>(DEFAULT_CONFIG, loadedConfig) as unknown as Configuration;
```

Comment:
as unknown as Configuration? Can we not only use the generics to type this correctly?

Reply:
Yes, I thought so too, But mergeDeepRight returns some other type. `MergeDeep<Configuration, Configuration>`, so had to type cast it here.